tests/kconfig/Kconfig

```diff
@@ -0,0 +1,22 @@
+menu "Application"
+
+menuconfig KCONFIG_APP_MSG_1
+    bool "Configure message 1"
+    default y
+    help
+        "This will enable configuring the first message via Kconfig"
```

Comment:
No need to put quotes around help texts. The end of the help text is determined by looking for the first non-empty line that's less indented than the first line.